source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```diff
@@ -0,0 +1,144 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "client.watch succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [aggregate], errorCode: 11600 }
+        operation: &retryable_operation
+            name: watch
+            object: client
+            result: []
+        expectations:
+            - &retryable_command_started_event
+                command_started_event:
+                    command:
+                        aggregate: *collection_name
+                        cursor: {}
+                        pipeline:
+                          - $changeStream:
+                                fullDocument: default
```

Comment:
This seems like a reasonable place to inline the expected pipeline if it wouldn't make the line too long. I think a good rule of thumb is to inline things like documents and lengthy command arguments, which aren't terribly important to the spec itself.